packages/helm/symphony/templates/redis-configmap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: redis-config-map
+data:
+  redis.conf: |
+    protected-mode yes
```

Comment:
Hi, Haishi. sidecar would only support single replica mode for better security. This configMap is only loaded for sidecar mode

Reply:
After a second thought, OSS could use secret auth as a better security option so I will move this change to ADO repo as a workaround. I will abandon this PR.

Reply:
We need to support Redis clustering. I don't think enabling clustering as suggested in my comment produces worse security. The single replica setting might be okay for short-term, but for long term, we need to support Redis clustering for sure, otherwise when we scale out Symphony API, this provides the same behavior of an in-memory store as each sidecar Redis is isolated.